tutorials/3d/portals/editing_rooms_and_portals.rst

```diff
@@ -75,20 +112,20 @@ To keep drawcalls to a minimum, the system offers the option to automatically me
 Plane Simplification
 ^^^^^^^^^^^^^^^^^^^^
 
-In some cases, automatically generated convex hull bounds may contain a very large number of planes. This is not ideal because it slows down determining which room a camera or object is within. The system can optionally simplify hulls. The degree of simplification can be selected by the user, between ``0`` (no simplification) and ``1`` (maximum simplification). You can also override this value in individual rooms.
+In some cases, the convex hulls automatically generated for rooms may contain a very large number of planes, particularly if you use curved surfaces. This is not ideal because it slows down the system. This option can optionally simplify hulls. The degree of simplification can be selected by the user, between ``0`` (no simplification) and ``1`` (maximum simplification). You can also override this value in individual rooms.
 
 Portals
 ~~~~~~~
 
 Portal Active
 ^^^^^^^^^^^^^
 
-Portals can be turned on and off at runtime. This is especially useful if you have open and closing doors.
+Portals can be turned on and off at runtime. This is especially useful if you have opening and closing doors.
 
 Two Way
 ^^^^^^^
 
-Portals can either be two way or one way. One way portals may be useful for example to create windows that can be seen out of, but not seen into. This can help performance when viewing buildings from outdoors.
+Portals can either be two way or one way. The default two way portals are quicker to set up in most circumstances, but one way portals can be useful. For example you can use one way portals to create windows that can be seen out of, but not seen into. This can help performance when viewing buildings from outdoors.
```

Comment:
```suggestion
Portals can either be two-way or one-way. The default two-way portals are quicker to set up in most circumstances, but one way portals can be useful. For example you can use one way portals to create windows that can be seen out of, but not seen into. This can help performance when viewing buildings from outdoors.
```